pyglet/image/codecs/pil.py

```diff
@@ -76,9 +76,9 @@ def decode(self, file, filename):
         # tostring is deprecated, replaced by tobytes in Pillow (PIL fork)
         # (1.1.7) PIL still uses it
```

Comment:
Pillow version comes from PIL.__version__ 
PIL.version is a compatibilty thing with the original PIL package, frozen at 1.1.7
The latest pillow is 8.0.1, does not have tostring
The latest pillow 7.x series is 7.2.0 and it has tostring

Reply:
I seems to me like the new code is working for both version, while what we had before would throw an error if the `tostring` was not found